install/kubernetes/helm/istio/values.yaml

```diff
@@ -296,8 +296,8 @@ prometheus:
   ingress:
```

Comment:
I _think_ the plan is to disable ingress specs, in favor of ingressgateway. 

cc: @costinm

Reply:
@douglas-reid  thanks for your comment. If this is the plan, then `ingress` item needs to be removed and also update this installation options to make all clear: https://github.com/istio/istio.github.io/blob/master/content/docs/reference/config/installation-options/index.md

Reply:
@LillyWu agreed. I'm awaiting the dust to settle on ingressgateway refactoring.